config.yml

```diff
@@ -5,8 +5,13 @@ pipeline:
 - name: CountVectorsFeaturizer
   OOV_token: oov
   token_pattern: (?u)\b\w+\b
+- name: CountVectorsFeaturizer
+  analyzer: "char_wb"
+  min_ngram: 1
+  max_ngram: 4
```

Comment:
do we need to add any oov_token or token_pattern to this CVF? or not because it's not looking at whole words as tokens

Reply:
nah i don't think we need to